src/Raven.Server/RavenServer.cs

```diff
@@ -2207,8 +2207,15 @@ private async Task<(Stream Stream, X509Certificate2 Certificate)> AuthenticateAs
                         // SSL, because that generate a nicer error for the user to read then just aborted
                         // connection because SSL negotiation failed.
                         true);
-
-                await sslStream.AuthenticateAsServerAsync(Certificate.Certificate, true, TcpUtils.SupportedSslProtocols, false);
+                await sslStream.AuthenticateAsServerAsync(new SslServerAuthenticationOptions
+                {
+                    ServerCertificate = Certificate.Certificate,
+                    ClientCertificateRequired = true,
+                    AllowRenegotiation = false,
```

Comment:
This by default is true. Do we want to change this? https://github.com/dotnet/runtime/blob/516956a01fee9aa732b199012a6ea0bcc2bd789f/src/libraries/System.Net.Security/src/System/Net/Security/SslServerAuthenticationOptions.cs#L16-L22

Reply:
Probably don't want to change this, you're right. I don't think we want to interfere with their logic.
https://github.com/dotnet/runtime/blob/6f6c94dbb2259deb083eefb755698cc8e0f0a4ce/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs#L216